src/SFA.DAS.QnA.Api.Types/Page/Input.cs

```diff
@@ -5,6 +5,7 @@ namespace SFA.DAS.QnA.Api.Types.Page
     public class Input
     {
         public string Type { get; set; }
+        public string ErrorMessage { get; set; }
```

Comment:
Could this be a little more descriptive, something like `TypeValidatorErrorMessage` or somert?